book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    # @title and @author instance variables
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] The Book class should have `@rentals` array in the constructor and add it to the `attr_accessor`:

  ```
  attr_accessor :title, :author, :rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end
  ```

- [ ] As you have added the `add_rental` method in the Person class kindly do the same in the Book class. Be sure to take note of the order of the arguments of the rental object.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```